Uplift/AppDelegate.swift

```diff
@@ -26,6 +26,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         setupGoogleSignIn()
         
         let defaults = UserDefaults.standard
+        defaults.set(false, forKey: Identifiers.hasSeenOnboarding)
```

Comment:
testing artifact? won't this force onboarding to show each time?

Reply:
o shoot good catch